src/darker/tests/test_import_sorting.py

```diff
@@ -1,3 +1,8 @@
+from pathlib import Path
+from textwrap import dedent
+
+import pytest
```

Comment:
I'd prefer all imports at the top for clarity unless a local import is required to avoid import loops.

```suggestion
import pytest
from black import find_project_root
```